src/util/parseoptions.cpp

```diff
@@ -17,13 +17,34 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/cmdline.h>
 #include <util/parseoptions.h>
 #include <util/signal_catcher.h>
+#include <boost/program_options.hpp>
 
+using namespace boost::program_options;
```

Comment:
don't use `using namespace`, always use the full scope.

Reply:
ok, done.